sdk/lusid_drive/utilities/utility_functions.py

```diff
@@ -29,3 +35,132 @@ def get_file_id(api_factory, file_name, folder_id):
     file_id = name_to_id(response, file_name)
 
     return file_id
+
+
+def create_folder(api_factory, folder_path, folder_name):
+    """
+    This function creates a new folder on LUSID Drive.
+
+    param api_factory ApiClientFactory: A LUSID Drive API Factory
+    param folder_path str: The folder path on LUSID Drive
+    param folder_path str: The new folder name on LUSID Drive
+
+    returns: a CreateFolder responses
+
+    """
+
+    folder_api = api_factory.build(lusid_drive.api.FoldersApi)
+
+    try:
+
+        create_folder_request = folder_api.create_folder(
+            create_folder=models.CreateFolder(folder_path, folder_name))
+
+        return create_folder_request
+
+    except lusid_drive.ApiException as e:
+
+        if json.loads(e.body)["code"] == 664:
+
+            return json.loads(e.body)["detail"]
+
+
+def create_nested_folder(api_factory, folder_path):

```

Comment:
Could be worth adding a path length check similar to 
https://github.com/finbourne/drive-sdk-csharp-preview/blob/master/sdk/Lusid.Drive.Sdk/Utilities/FoldersApiExtensions.cs#L39

unlikely it will ever be hit, but always good to be safe